Source/sdc/vode_rhs_true_sdc.H

```diff
@@ -76,12 +76,8 @@ void rhs(const Real time, burn_t& burn_state, dvode_t& vode_state, RArray1D& dUd
 
 template<class MatrixType>
 AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE
-void jac (burn_t& burn_state, dvode_t& vode_state, MatrixType& pd)
+void jac(const Real time, burn_t& burn_state, dvode_t& vode_state, MatrixType& pd)
```

Comment:
The space after the function name is intentional, it's from the Weiqun style guide :-D

Reply:
restored